src/oldbot/RobotPlayer.java

```diff
@@ -32,14 +32,12 @@ public static void run(RobotController rc) throws GameActionException {
 
         turnCount = 0;
 
-        System.out.println("I'm a " + rc.getType() + " and I just got created!");
```

Comment:
I also removed all the System.out.println because they go into the playtest report.